packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/lib/stores.ts

```diff
@@ -0,0 +1,59 @@
+import { DRUPAL_URL } from './constants';
+import { DrupalState } from '@pantheon-systems/drupal-kit';
+
+// For each locale, make an instance of DrupalState (LocaleStore)
+export const makeLocaleStores = ({ locales, debug = false } : {locales: string[], debug: boolean}) =>
+	locales.length > 1
+		? locales.map(
+				(locale) =>
+					new DrupalState({
+						apiBase: DRUPAL_URL,
+						defaultLocale: locale,
+						debug: debug,
+						clientId: process.env.CLIENT_ID,
+						clientSecret: process.env.CLIENT_SECRET,
+					}),
+		  )
+		: [
+				new DrupalState({
+					apiBase: DRUPAL_URL,
+					debug: debug,
+					clientId: process.env.CLIENT_ID,
+					clientSecret: process.env.CLIENT_SECRET,
+				}),
+		  ];
+
+// exporting so we can reuse in
+// getStaticProps and getServerSideProps
+/**
+ * @type {DrupalState[]}
```

Comment:
This is a leftover from some JSDoc typing. Whenever you see these 3 dots under the start of a function definition, its essentially a linting error for the JS/TS doc comment. You can hover over it and it tells you the error and suggests fixes. 

Make sure to update the comments to TSDoc when doing the TS conversion since the JSDoc comments are slightly different (yet another fun JavaScript ecosystem fact 🙃)
Sometimes I use https://tsdoc.org/play/ to make sure my comments display how I expect
![Screen Shot 2023-06-22 at 2 58 27 PM](https://github.com/pantheon-systems/decoupled-kit-js/assets/63015754/c43fb16d-67a4-4e80-91d7-ca4376cacdda)